tests/tests/test-xcm/test-mock-dmp.ts

```diff
@@ -78,3 +79,153 @@ describeDevMoonbeam("Mock XCM - receive downward transfer", (context) => {
     expect(alith_dot_balance).to.eq(10n * RELAY_TOKEN);
   });
 });
+
+describeDevMoonbeam("Mock XCMP - test XCMP execution", (context) => {
+  it("Should test DMP on_initialization and on_idle", async function () {
+    const metadata = await context.polkadotApi.rpc.state.getMetadata();
+    const balancesPalletIndex = (metadata.asLatest.toHuman().pallets as Array<any>).find(
+      (pallet) => {
+        return pallet.name === "Balances";
+      }
+    ).index;
+    let ownParaId = (await context.polkadotApi.query.parachainInfo.parachainId()) as any;
+
+    let numMsgs = 50;
+    // let's target half of then being executed
+
+    // xcmp reserved is BLOCK/4
+    let totalDmpWeight =
```

Comment:
about the `let` usage - it doesn't seem that we are overriding the values afterwards - maybe it should be `const` instead - also in a few variables below.